src/language-js/print/function-parameters.js

```diff
@@ -223,8 +223,16 @@ function shouldGroupFunctionParameters(functionNode, returnTypeDoc) {
   );
 }
 
+function shouldBreakFunctionParameters(functionNode) {
+  const { params } = functionNode;
```

Comment:
please use `getFunctionParameters`

Reply:
I changed it. Thank you for your review.